akvo/rsr/models/organisation.py

```diff
@@ -364,6 +376,13 @@ def support_partners(self):
         in common with, excluding self"""
         return self.published_projects().support_partners().exclude(id__exact=self.id)
 
+    def field_partners(self):
+        """
+        Returns an Organisation queryset of field partners of which self has at least
+        one project in common with.
+        """
+        return self.all_projects().field_partners().distinct()
```

Comment:
```Organisation.support_partners()``` uses ```self.published_projects()```. It seems logical to use the same in both methods, although I can't say if ```published_projects()``` or ```all_projects()``` is the correct one.